src/main/java/chess/domain/ChessGame.java

```diff
@@ -30,4 +31,18 @@ private void changeTurnColor() {
     public Map<Position, PieceType> collectBoard() {
         return moveState.collectBoard();
     }
+
+    public StatusDto calculateScore() {
+        return new StatusDto(moveState.calculateScore(Color.WHITE), moveState.calculateScore(Color.BLACK));
+    }
```

Comment:
```suggestion
    public StatusDto calculateScore() {
        return new StatusDto(
            moveState.calculateScore(Color.WHITE),
            moveState.calculateScore(Color.BLACK)
        );
    }
```
줄바꿈을 활용해서 가독성을 높여보는건 어떨까요?

Reply:
좋은것 같습니다. 가독성이 훨신 좋네요!